### ISSUE TITLE
OverflowSet A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#7761]
Tracking all of the issues brought up with the 1/22 accessibility pass on OverflowSet control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Be4af1f95-30ff-43e8-9dfc-09de1866ce7f%7D&action=default&uid=%7BE4AF1F95-30FF-43E8-9DFC-09DE1866CE7F%7D&ListItemId=1700&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 
  
 - [x] Bug1: [Accessibility][Programmatic access]: Buttons must have discernible text." 
 -  [x] Bug2: [Accessibility][Programmatic access]: Incorrect Aria-label is defined for 'Add' button. " 
  - [x] Bug3: [Accessibility][Programmatic access]: Narrator does not read for tooltip as it is not properly defined in code.  
  - [ ] Bug4: [Accessibility][Usable]: Controls under 'OverflowSet Custom Example' section have multiple scan mode focuses.  
  - [x] Bug5: [Accessibility][Usable]: Narrator does not convey about role-menubar when user navigates the first control of the section.  
  - [x] Bug6: [Accessibility][Programmatic access]: User does not get the information related to orientation of the menus.  
  - [x] Bug7: [Accessibility][Programmatic access]: No position is defined for items inside the menubar.  
 
 
 Component: OverflowSet  
@micahgodbolt

**Comment by micahgodbolt** [comment]
3 - removed tooltips as they aren't important for the demo

**Comment by micahgodbolt** [comment]
1 - added name values to buttons without visible text

**Comment by micahgodbolt** [comment]
2  - changed label text to match

**Comment by micahgodbolt** [comment]
6 - added aria-orientation vertical/horizontal

**Comment by micahgodbolt** [comment]
5 - narrator does not convey any information about menubars. This w3.org example provides the exact same result:

https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/menubar/menubar-1/menubar-1.html

**Comment by micahgodbolt** [comment]
7 - in the same menubar example, narrator is not reading off the positions either. https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/menubar/menubar-1/menubar-1.html

From everything i've read, posinset is only required if not all of the items are present in the DOM. 

https://www.digitala11y.com/aria-posinset-properties/
